cypress.json

```diff
@@ -7,6 +7,6 @@
   "video": true,
   "projectId": "kobqo4",
   "key": "6ecaeca9-8292-49d5-b954-f5d330526703",
-  "pageLoadTimeout": 180000,
-  "defaultCommandTimeout": 15000
+  "pageLoadTimeout": 200000,
+  "defaultCommandTimeout": 20000
 }
```

Comment:
Why did you need to change these? They are already bigger than they should be...